app/payments.py

```diff
@@ -68,20 +50,126 @@ def list_payments():
+              description: type of payment (credit, debit, paypal)
+              example: credit
+            details:
+              schema:
+                id: Detail
+                properties:
+                  user_name:
+                    type: string
+                    description: user name associated with payment method
+                    example: Jimmy Jones
+                  expires:
+                    type: string
+                    description: credit/debit card expiration date (mm/yyyy)
+                    example: 02/2020
+                  card_number:
+                    type: string
+                    description: 16-digit credit/debit card number
+                    example: 1111222233334444
+                  card_type:
+                    type: integer
+                    description: type of credit/debit card (Visa, Mastercard, etc.)
+                    example: Visa
+    responses:
+      201:
+        description: Payment created
+        schema:
+          type: array
+          items:
+            schema:
+              id: Payment
+              properties:
+                id:
+                  type: integer
+                  description: id auto-generated by database
+                nickname:
+                  type: string
+                  description: user-made nickname for payment method
+                user_id:
+                  type: integer
+                  description: foreign key id for user
+                payment_type:
+                  type: string
+                  description: type of payment (credit, debit, paypal)
+                is_default:
+                  type: boolean
+                  description: signifies default payment method for purchases
+                  default: false
+                is_removed:
+                  type: boolean
+                  description: soft delete for removed payments with pending transactions
+                  default: false
+                charge_history:
+                  type: float
+                  description: history of purchase amounts for specified payment method
+                  default: 0.0
+                details:
+                  schema:
+                    id: details
+                    properties:
+                      user_name:
+                        type: string
+                        description: user name associated with payment method
+                      expires:
+                        type: string
+                        description: credit/debit card expiration date (mm/yyyy)
+                      card_number:
+                        type: string
+                        description: 16-digit credit/debit card number
+                      card_type:
+                        type: integer
+                        description: type of credit/debit card (Visa, Mastercard, etc.)
+
+      400:
+        description: Bad Request (the posted data was not valid)
+    """
```

Comment:
Hmm.  Is it required by the packages we're using to place the swagger documentation in a docstring like this?  Usually swagger specs are written in a separate `.yaml` file.  It would be much easier to maintain things that way.

There is a guide here: http://brunorocha.org/python/flask/flasgger-api-playground-with-flask-and-swagger-ui.html

I suggest that we go with putting everything in a `.yaml` file because it will make maintaining it much easier.  Not to mention, many popular editors will offer better support since they'll pick up on the filetype and enforce the proper rules.